lang/cs/Org.Apache.REEF.Bridge/Org.Apache.REEF.Bridge.nuspec

```diff
@@ -1,4 +1,20 @@
 <?xml version="1.0"?>
```

Comment:
Cannot pass rat checking as pom file under lang/cs folder has been removed. We need to put that pom file back with profile removed so that that pom file  can inherit exclusions from parent's pom file.